packages/backend/src/controllers/flash.ts

```diff
@@ -0,0 +1,8 @@
+import { RequestHandler } from 'express';
+
+export const getFlash: RequestHandler = (req, res) => {
+    res.json({
+        status: 'ok',
+        results: req.flash(),
+    });
+};
```

Comment:
I wouldn't consider this a controller 🤔  Are we planning to use it across multiple routes? Just seems like this code could be in the route.

Reply:
Moving this and `userController` to inline in routers